client/src/pages/Login.js

```diff
@@ -1,150 +1,158 @@
 import React from 'react';
-import {
-    withStyles,
-    Button
-} from '@material-ui/core';
-import {Link} from 'react-router-dom';
+import { withStyles, Button } from '@material-ui/core';
+import { capitalize } from '@material-ui/core/utils';
+import { Link } from 'react-router-dom';
```

Comment:
we might have to come to an agreement on more coding styles?

Reply:
Hmm this was done by prettier